### PR TITLE
Create LICENSE.txt

### PR DESCRIPTION
**Pull request by lucasgonze**
Fixes https://github.com/oprogramador/deep-equal-in-any-order/issues/76#

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 oprogramador
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
```
Comment (oprogramador) [inline LICENSE.txt:5]: Sorry for a delay but it seems that this text differs from https://opensource.org/licenses/MIT .

I didn't have enough time to find out the exact differences.

Comment (lucasgonze) [reply LICENSE.txt:5]: Yes, I'm aware of the difference. It is the addition of the one line at the top: "MIT License". The text at https://opensource.org/licenses/MIT doesn't have it. The version with the additional line was provided by GitHub. It's their standard template for the MIT License.

Comment (oprogramador) [reply LICENSE.txt:5]: I see also a different division into lines.

**Comment by oprogramador** [comment]
I close this PR as the license is now added and published to NPM in `deep-equal-in-any-order@2.0.3`.

My version is easier to compare with https://opensource.org/licenses/MIT as I can just copy-paste these 3 lines, and check whether they're exactly the same.

**Comment by oprogramador** [comment]
Anyway, thanks for your contribution.

**Comment by lucasgonze** [comment]
Thanks for implementing this. I really appreciate it.

**Pull request closed without merging**